### ISSUE TITLE
Prompt for missing credentials and cache them

### ISSUE THREAD

**Issue by kris-nova** [kris-nova/kubicorn#303]
In an attempt to make kubicorn even easier to use can we prompt the user for credentials if they are not configured properly?

For instance if we need an API key and we can't find it anywhere simply prompt the user for input (and maybe even offer a link to documentation on where to find it).. and we have validated that new secret works.. can we cache to disk and make `kubicorn` smart enough to always check the cache as well.

**Comment by xmudrii** [comment]
@kris-nova I agree on this one but about last part. Is it smart idea to cache credentials our self?
It's okay to ask user for credentials and link them to docs, but does we want to take it on ourself to store it? You know the error, it will not fail, but we will let you store wherever you want. :)

I had same idea for SSH and SCP - create `sshagent` package and store key in in-memory agent (if system one is not available). But once command finish, it will be lost.

**Comment by prateekgogia** [comment]
We shouldn't cache the credentials, rather create it as a secret in Kubernetes ? If that can help.

**Comment by kris-nova** [comment]
Being that we are in the creating kubernetes from scratch biz - there might not always be a kubernetes cluster available to store it in
